### DISCUSSION TITLE
Git-based blogging

### DISCUSSION THREAD

**Discussion by tianhuat** [quasarframework/quasar#11201] [General - Components / Directives / etc]
Hi does Quasar supports Git Based Blogging (i.e., read the markdown static files and render the blog)? If yes, how to achieve that?

**Comment by hawkeye64** [comment]
Look at the Quasar `docs` folder in the repo to see how it's done.

**Comment by tianhuat** [comment]
Thanks, but I can't find the exact page that discussed this. To be more specific, what I am referring to is the feature in the following:
https://webdevetc.com/blog/gridsome-markdown-blog/
https://nuxtjs.org/tutorials/creating-blog-with-nuxt-content/

**Comment by hawkeye64** [comment]
It's not in the docs, it is how the docs are made, using markdown.
Here is a good starting point: https://github.com/quasarframework/quasar/blob/dev/docs/build/md.js
